libvast/src/format/csv.cpp

```diff
@@ -188,6 +189,7 @@ const char* reader::name() const {
 
 caf::optional<record_type>
 reader::make_layout(const std::vector<std::string>& names) {
+  VAST_TRACE(__func__, VAST_ARG(names));
```

Comment:
`__func__` is the first parameter of all trace logs by default, no need to specify it here.